tests/keras/layers/convolutional_test.py

```
@@ -223,11 +227,32 @@ def test_conv2d_transpose():
+    # to stride
+    with pytest.raises(ValueError):
+        model = Sequential([convolutional.Conv2DTranspose(
+            filters=filters,
+            kernel_size=3,
+            padding=padding,
+            output_padding=(0, 3),
+            strides=(1, 3),
+            batch_input_shape=(None, num_row, num_col, stack_size))])
+    # Output padding greater than stride
+    with pytest.raises(ValueError):
+        model = Sequential([convolutional.Conv2DTranspose(
```

Ruff (0.12.2)

Local variable `model` is assigned to but never used

Remove assignment to unused variable `model`

(F841)